system/os_host/library/spf/spf.c

```diff
@@ -160,6 +160,7 @@ int sfp_printi(char **out, int i, int b, int sg, int width, int pad, int letbase
 	return pc + sfp_prints (out, s, width, pad);
 }
 
+#ifndef WIN32
```

Comment:
Could you please revert those additions and exclude spf completely from windows build?

os_platform/common/source/framework/framework.cpp:
```c
#if !defined(__APPLE__) && !defined(WIN32)
#include "../../../os_host/library/spf/spf.h"
#include "../../../os_host/library/spf/spf.c"
#endif
```

Reply:
Done !